src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -688,6 +707,32 @@ class ExposureSubmissionCoordinator: NSObject, ExposureSubmissionCoordinating, R
 
 		navigationController?.present(alert, animated: true)
 	}
+	
+	// MARK: Key Submission Service
+	
```

Comment:
Please Add Unit Tests ;)

Reply:
Created a separate Task for Unit Testing.